azure-nextgen-static-website/index.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as cdn from "@pulumi/azure-nextgen/cdn/latest";
+import * as resources from "@pulumi/azure-nextgen/resources/latest";
+import * as storage from "@pulumi/azure-nextgen/storage/latest";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+
+// TODO: Remove after autonaming support is added.
+const randomSuffix = new random.RandomString("random", {
+    length: 10,
+    special: false,
+    upper: false,
+});
+
+const resourceGroup = new resources.ResourceGroup("resourceGroup", {
+    resourceGroupName: pulumi.interpolate `rg${randomSuffix.result}`,
+    location: "westus",
```

Comment:
No need to hard-code the location

Reply:
If I exclude resourceGroup creation fails. Switched it to a config entry with a default instead.